layers/landcover/style.json

```diff
@@ -18,19 +18,7 @@
       },
       "paint": {
         "fill-color": "rgba(255,255,255,1)",
-        "fill-opacity": {
-          "base": 1,
-          "stops": [
-            [
-              0,
-              0.9
-            ],
-            [
-              10,
-              0.3
-            ]
-          ]
-        }
+        "fill-opacity": 0.9
```

Comment:
Do we want all styles to be fully opaque (e.g. `1`)? Probably it makes little difference either way. 🤷

Reply:
I set up a slight opacity because I've seen occurrences of the landcover that overlap. Without this opacity, they would not stand out as they do in the current implementation. :shrug: